src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -25,7 +25,7 @@ public class ModuleClass implements Identity<ModuleClass>, Comparable<ModuleClas
 
     private final String className;
 
-    private final List<Session> sessions;
+    private final ArrayList<Session> sessions;
```

Comment:
Perhaps we should use `List` instead of `ArrayList` for better abstraction?

Reply:
I think it's okay for it to be defined as `ArrayList` internally, as previously accepting just `List` allowed a `List` from another class to be passed instead, which may lead to undesirable behaviour.